agate/table/print_table.py

```diff
@@ -140,7 +160,6 @@ def write_row(formatted_row):
 
         write('%s%s%s' % (v_line, text, v_line))
 
-    # Dashes span each width with '+' character at intersection of
```

Comment:
This is my favorite kind of comment: a bit too obvious, and therefore ignored by many people, including the person who changed the code to make it no longer true.